### PR TITLE
 Initial swift codestyle

### PR DESCRIPTION
**Pull request by kostiakoval**
Added some initial code style. Check it you have any comments or things you disagree :)

### CODE REVIEW

File: code-style/swift.md [hunk 1]
```diff
@@ -1 +1,149 @@
...
+```
+
+### Minimal Imports
+
+Keep imports minimal. For example, don't import `UIKit` when importing `Foundation` will suffice.
+
+
+## Spacing and Indentation
+
+* Indent using 4 spaces rather than tabs to conserve space and help prevent line wrapping. This should be configured on the project.
```
Comment (hfossli) [inline code-style/swift.md]: Is this xcode default?

Comment (kostiakoval) [inline code-style/swift.md]: yes

**Comment by kostiakoval** [comment]
The only thing i dislike is - it's to verbose. I would love to keep the Style guide as simple as possible, but at the same time informative enough. 
I think we can improve it in the future.

**Comment by hfossli** [comment]
What about using collapsable sections like we do in https://github.com/agens-no/Anymotion readme  (see installation section)?

**Comment by kostiakoval** [comment]
We could add this as well. Is there a way to collapse/ expand all of them at once?

I will add TOC later, like here - 
https://github.com/raywenderlich/swift-style-guide/blob/master/README.markdown

**Comment by hfossli** [comment]
> We could add this as well. Is there a way to collapse/ expand all of them at once?

No, not that I know of.

**Comment by kostiakoval** [comment]
We can changes and improve formatting later. The most important is the content. So let's all have quick look and vote/comment: @jalmaas @hfossli @aarsland  @osunnarvik.

**Comment by kostiakoval** [comment]
Don't see any disagreement on this and that's why I'm going to merge it. 

as a general practice is someone finds something he dissagree with, he can start a discussion afterwards.

**Pull request merged**